### ISSUE TITLE
Completed Kata; solutions still locked

### ISSUE THREAD

**Issue by SpecterTerrasbane** [Codewars/codewars.com#582]
Completed Kata: https://www.codewars.com/kata/simple-encryption-number-2-index-difference in Python.

Shows up in my profile's Completed list at https://www.codewars.com/users/kjmosher .

Still getting "Solutions have been withheld" message when attempting to view Kata's solutions.

Attempted to resubmit solution; Basic tests passed, attempted final tests.  Output window freezes on "Executing" , but allowed me to submit ..  Took me right back to the "Solutions have been withheld" screen.

**Comment by SpecterTerrasbane** [comment]
Same thing just happened on kata: https://www.codewars.com/kata/sum-up-the-random-string

**Comment by kb0rg** [comment]
I had the same issue on https://www.codewars.com/kata/sum-the-repeats, over a month ago.

**Comment by lvsz** [comment]
Just had the same issue with https://www.codewars.com/kata/create-phone-number, clicked on "forfeit eligibility" after submitting and getting accepted twice, and my code wasn't there. Submitted it a third time, and it's still not there.

**Comment by SlyBeetle** [comment]
I submitted [my solution](https://www.codewars.com/kata/grasshopper-grade-book/train/csharp/58443a0d101928a1bc0000a9) two times, but got on "Solutions" tab:
"Solutions have been withheld

Since you have not yet solved this kata we have hidden the solutions from you. If you choose to view the solutions you will forfeit your eligibility to earn honor/rank progress for this kata."

**Comment by jhoffner** [comment]
You can use preview.codewars.com. The issue is fixed there. It will be released to www next week along with a big update.

**Comment by kazk** [comment]
@jhoffner was this issue resolved?

---

Similar issue: #314, #441, #553, #617, #665

**Comment by Sinelife** [comment]
Same thing with Katas
https://www.codewars.com/kata/5596f6e9529e9ab6fb000014
https://www.codewars.com/kata/557f6437bf8dcdd135000010

**Comment by Sinelife** [comment]
Completed this Katas
https://www.codewars.com/kata/5596f6e9529e9ab6fb000014
https://www.codewars.com/kata/557f6437bf8dcdd135000010

They added to my list of finished in my status. But I didn't get honor for them and they didn't mark with tick in list of all Katas(All other my finished Katas marked with tick).

**Comment by lbvf50mobile** [comment]
Time to time I have to duplicate sending correct solutions get solutions list open. Because after fist successful `Submit Final` a solutions stays closed. Today this happens twice on the [baby-shark-lyrics-generator](https://www.codewars.com/kata/baby-shark-lyrics-generator/train/ruby) kata when I solve it in `JS` and `Ruby`.  Also very often need to click `Attempt` several times before `Submit Final` appears.  
#617 #1683 - relates issues.

**Comment by Steffan153** [comment]
@lbvf50mobile That's https://github.com/Codewars/codewars.com/issues/1683

**Comment by falcowinkler** [comment]
I had the same issue today. I think i got the honour points twice. 
https://www.codewars.com/kata/a-string-of-sorts/train/haskell

**Comment by ArtStar01** [comment]
Same here. Is there are any updates?

<img width="1243" alt="Screen Shot 2019-08-23 at 12 14 18 PM" src="https://user-images.githubusercontent.com/6277069/63582791-63212080-c5a2-11e9-9150-f8f1424e653a.png">
<img width="1042" alt="Screen Shot 2019-08-23 at 12 14 39 PM" src="https://user-images.githubusercontent.com/6277069/63582794-65837a80-c5a2-11e9-9128-30affe4a1a14.png">

**Comment by falcowinkler** [comment]
i don't mind the locked solutions bug at all, i got in the habit of backing up my solutions. but i would really appreciate not getting the extra honour / rank, that's just unfair

**Comment by FArekkusu** [comment]
@falcowinkler, it didn't "exist for 3 years". The bug reappeared a few months ago.

**Comment by kazk** [comment]
No, it exists for at least 3 years. But the frequency seemed to have increased this year, either because of updating the database driver at the end of last year or the increased amount of data.
Also, there are many other symptoms caused by the same bug that's been reported (solution not shown, solution locked, solution appears multiple times under profile, failing to submit after passing, not getting honor, getting honor multiple times, etc).

Codewars has many bugs that existed for many years. This is because no one was working on the project for few years until I started to work on it part time last year (spent most of the time just updating EOLed components from 6 years ago :().

This is all because we needed to focus on Qualified which is required for Codewars to exist. We've reached the point that we can start working on Codewars more so we'll make a post explaining the current situation and our future plans soon.
